crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -189,12 +189,16 @@ class MotionPlanningServer : public rclcpp::Node
     trajopt_surface_config.surface_coeffs = surface_coeffs;
     trajopt_surface_config.waypoints_critical = false;
     trajopt_surface_config.longest_valid_segment_fraction = 0.1;
-    trajopt_surface_config.special_collision_constraint.push_back({ "eoat_link", LOADED_PART_LINK_NAME, -0.02, 15.0 });
-    trajopt_surface_config.special_collision_constraint.push_back(
-        { "sander_center_link", LOADED_PART_LINK_NAME, -0.02, 15.0 });
+    //    trajopt_surface_config.special_collision_constraint.push_back({ "eoat_link", LOADED_PART_LINK_NAME,
+    //    -0.02, 15.0 }); trajopt_surface_config.special_collision_constraint.push_back(
+    //        { "sander_center_link", LOADED_PART_LINK_NAME, -0.02, 15.0 });
+    trajopt_surface_config.special_collision_constraint.push_back({ "eoat_link", "table", -0.05, 15.0 });
```

Comment:
These seem like parameters that should be made configuration

Reply:
I will make a yaml config file for all of the motion planning params, will make testing in the future easier.